aikau/src/main/resources/alfresco/preview/PdfJs/DocumentView.js

```diff
@@ -309,10 +310,10 @@ define(["dojo/_base/declare",
          {
             var currentPage = this.pages[0],
                 container = currentPage.container,
-                hmargin = parseInt(domStyle.get(container, "margin-left")) + parseInt(domStyle.get(container, "margin-right")),
-                vmargin = parseInt(domStyle.get(container, "margin-top")),
-                contentWidth = parseInt(currentPage.content.pageInfo.view[2]),
-                contentHeight = parseInt(currentPage.content.pageInfo.view[3]),
+                hmargin = parseInt(domStyle.get(container, "margin-left"), 10) + parseInt(domStyle.get(container, "margin-right"), 10),
+                vmargin = parseInt(domStyle.get(container, "margin-top"), 10),
+                contentWidth = parseInt(currentPage.content.pageInfo.view[2], 10),
+                contentHeight = parseInt(currentPage.content.pageInfo.view[3], 10),
```

Comment:
Is it worth updating the .jshintrc file to pick up this (and other?) errors prior to commit, rather than wait for a sonar run in future?